pkg/webui/console/views/device-add/manual/connect.js

```diff
@@ -32,11 +37,14 @@ const mapStateToProps = state => ({
   mayEditKeys: checkFromState(mayEditApplicationDeviceKeys, state),
   prefixes: selectJoinEUIPrefixes(state),
   createDevice: (appId, device) => api.device.create(appId, device),
+  generateDevEUI: appId => api.application.generateDevEUI(appId),
```

Comment:
See my other comment. Define this within the component rather, using `useCallback`.